js/mcqModel.js

```diff
@@ -168,10 +168,59 @@ define([
                 _selectedItems: [],
                 _isAtLeastOneCorrectSelection: false
             });
+        },
+
+        getInteractionObject: function() {
+            var interactions = {
+                correctResponsesPattern: [],
+                choices: []
+            };
+
+            interactions.choices = this.get('_items').map(function(item) {
+                return {
+                    id: (item._index + 1).toString(),
+                    description: item.text
```

Comment:
i don't think we're trying to implement xapi throughout the course, in the same way we didn't implement scorm. we just need to provide apis for the appropriate information from each plugin. passing the language with every interaction.choices seems overkill, especially as the xapi plugin can derive that itself.